documentation/sphinx/source/tls.rst

```diff
@@ -106,31 +94,29 @@ The default peer verification is ``Check.Valid=0``.
 Default Password
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-There is no default password. If no password is specified, the plugin assumes that private key is unencrypted.
+There is no default password. If no password is specified, TLS assumes that private key is unencrypted.
```

Comment:
I read "the plugin" to mean "our code" and "TLS" to mean "the Transport Layer Security standard", which makes this sentence read... not quite right.  I think I'd recommend changing the phrasing to "If no password is specified, it is assumed that the private key is unencrypted", and similar rephrasing to the rest of the s/the plugin/TLS/ changes that you've done.  If you need to name something, I think it'd be more proper to name `FoundationDB` rather than TLS.